dataprofiler/tests/data_readers/test_data.py

```diff
@@ -14,6 +16,60 @@ def test_read_data(self):
         for data_type, none_type in zip(data_types, none_types):
             Data(data=none_type, data_type=data_type)
 
+class TestDataReadFromURL(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        test_dir = os.path.join(test_root_path, 'data')
+        cls.input_file_names = [
+            dict(path=os.path.join(test_dir, 'csv/diamonds.csv'),
+                 count=53940, delimiter=',', has_header=[0],
+                 num_columns=10, encoding='utf-8', data_type='csv'),
+            dict(path=os.path.join(test_dir, 'avro/users.avro'), count=4,  data_type='avro'),
+            dict(path=os.path.join(test_dir, 'json/iris-utf-16.json'), encoding='utf-16', count=150, data_type='json'),
+            dict(path=os.path.join(test_dir, 'parquet/iris.parq'), count=150, data_type='parquet'),
+            dict(path=os.path.join(test_dir, 'txt/code.txt'), count=150, data_type='text'),
```

Comment:
Format these entries similar to the first one so that they all are under 80 chars